Rmds/r_and_rstudio.Rmd

```diff
@@ -0,0 +1,78 @@
+# R and RStudio {-}
+
+To complete the installs of R and RStudio, you must have administrator privileges on your machine. Choose the version appropriate for your operating system. You can also watch a video for a [Mac](https://youtu.be/Oy9BPxt8gQ0?t=270) or a [Windows](https://youtu.be/7f9ADV2v7CI?t=336) install (you only need to install R and RStudio, and none of the other programs mentioned in the videos). Please also note that the versions of R and RStudio in the video are older. Instead, install or update to the latest version supported by your operating system.
+
+1. R https://cran.r-project.org/
+    - version `r getRversion()` (only for macOS Yosemite or older: legacy version `r params$r_legacy_version`)
+    - Choose among "Download and Install R" section (**not** "Source Code for all Platforms").
+    
+        **Note for Mac users:** If you have an M1 (and higher) Mac, you **must** install the Intel version of R (Bioconductor packages installed below are not yet compatible with the arm64 version).
+        
+    - You **must** have R installed before proceeding to step 2.
+1. RStudio https://rstudio.com/products/rstudio/download/#download
+    - Version `r params$rstudio_version` (only for macOS El Capitan and older: legacy version `r params$rstudio_legacy_version`)
+    - Select FREE Desktop version.
+    - Choose among "Installers for Supported Platforms" section (**not** "Zip/Tarballs") or follow the link to older versions of RStudio if you need a legacy version.
+1. In RStudio:
+    - Select the "Console" tab of the left panel ![](`r here("images", "one.png")`){width=3%}.
+    - Copy the following code after the prompt `>` ![](`r here("images", "two.png")`){width=3%} to install all necessary packages:
+ 
+        ```{r install-packages, eval = FALSE}
+        if (!require(tidyverse))    install.packages("tidyverse")
+        if (!require(pheatmap))     install.packages("pheatmap")
+        if (!require(RColorBrewer)) install.packages("RColorBrewer")
+        if (!require(BiocManager))  install.packages("BiocManager")
+        if (!require(DESeq2))       BiocManager::install("DESeq2")
+        ```
+
+        While installing (or updating, see below) packages, you might be prompted:
+
+        ```
+          There is a binary version available, but the source version is later:
+        
+        <table of package names>
+        
+        Do you want to install from sources the package which need compilation? (Yes/no/cancel)
+        ```
+
+        Always enter `no` followed by <kbd>enter</kbd>.
```

Comment:
Capital E on "enter"?

Reply:
I have used lowercase for all modifier/special keys elsewhere on the website. (I cannot decide what way I find better lower vs. uppercase and there does not seem to be a convention.)